packages/astro/src/@types/astro.ts

```diff
@@ -571,12 +571,7 @@ export interface AstroUserConfig {
 	 *
 	 * When using this option, all of your static asset imports and URLs should add the base as a prefix. You can access this value via `import.meta.env.BASE_URL`.
 	 *
-	 * By default, the value of `import.meta.env.BASE_URL` includes a trailing slash. If you have the [`trailingSlash`](https://docs.astro.build/en/reference/configuration-reference/#trailingslash) option set to `'never'`, you will need to add it manually in your static asset imports and URLs.
-	 *
-	 * ```astro
-	 * <a href="/docs/about/">About</a>
-	 * <img src=`${import.meta.env.BASE_URL}image.png`>
-	 * ```
+	 * The value of `import.meta.env.BASE_URL` is the same as `base`, unless `trailingSlash: "always"` or `trailingSlash: "never"` is set, which will ensure or remove a trailing slash from the `base` respectively.
```

Comment:
One query around "same as" — don't we add a leading slash to `base` if not present?
Maybe this is more accurate?
```suggestion
	 * The value of `import.meta.env.BASE_URL` respects your `trailingSlash` config and will include a trailing slash if you explicitly include one or if `trailingSlash: "always"` is set. If `trailingSlash: "never"` is set, `BASE_URL` will not include a trailing slash, even if `base` includes one.
```

Reply:
> One query around "same as" — don't we add a leading slash to `base` if not present?

Yeah that's true. I'd perhaps like to keep that not mentioned to not overload how `base` works, and because the docs always recommend with a leading slash.

Your suggestion sounds good to me though 👍